frontend/talentsearch/src/js/components/reviewMyApplication/ReviewMyApplicationPage.tsx

```diff
@@ -0,0 +1,266 @@
+  poolAdvertisement: PoolAdvertisement;
+  poolCandidateId: string;
+  closingDate: Date;
+  jobTitle: Maybe<string>;
+  poolId: string;
+}
+
+export const ReviewMyApplication: React.FunctionComponent<
+  ReviewMyApplicationProps
+> = ({
+  applicant,
+  poolAdvertisement,
+  poolCandidateId,
+  closingDate,
+  jobTitle,
+  poolId,
+}) => {
+  const intl = useIntl();
+  const directIntakePaths = useDirectIntakeRoutes();
+  const experiences = applicant.experiences?.filter(notEmpty) || [];
+  const missingSkills = {
+    requiredSkills: poolAdvertisement.essentialSkills?.filter(notEmpty),
+    optionalSkills: poolAdvertisement.nonessentialSkills?.filter(notEmpty),
+  };
+  const hasExperiences = notEmpty(applicant.experiences);
+  const { isProfileComplete } = applicant;
+  const isApplicationComplete =
+    isProfileComplete ||
+    getMissingSkills(
+      missingSkills.requiredSkills || [],
+      hasExperiences ? flattenExperienceSkills(experiences) : [],
+    ).length === 0;
+
+  return (
+    <ApplicationPageWrapper
+      closingDate={closingDate}
+      title={intl.formatMessage({
+        defaultMessage: "My application profile",
+        description: "Title for review application page.",
+      })}
+      crumbs={[
+        {
+          title: intl.formatMessage({
+            defaultMessage: "My applications",
+            description: "Breadcrumb for review application page.",
+          }),
+          href: directIntakePaths.allPools(),
+        },
+        {
+          title:
+            jobTitle || intl.formatMessage(commonMessages.jobTitleNotFound),
+          href: directIntakePaths.poolApply(poolId),
+        },
+        {
+          title: intl.formatMessage({
+            defaultMessage: "Step 1",
+            description: "Breadcrumb for review application page.",
+          }),
+        },
+      ]}
+      navigation={{
+        currentStep: 1,
+        steps: [
+          {
+            path: directIntakePaths.reviewApplication(applicant.id),
+            label: intl.formatMessage({
+              defaultMessage: "Step 1: Review my profile",
+              description: "Navigation step in sign and submit page.",
+            }),
+          },
+          {
+            path: directIntakePaths.signAndSubmit(applicant.id),
+            label: intl.formatMessage({
+              defaultMessage: "Step 2: Sign and submit",
+              description: "Navigation step in sign and submit page.",
+            }),
+            disabled: !isApplicationComplete ?? false,
+          },
+        ],
+      }}
+      subtitle={jobTitle || intl.formatMessage(commonMessages.jobTitleNotFound)}
+    >
+      <UserProfile
+        applicant={applicant}
+        sections={{
+          myStatus: { isVisible: false },
+          hiringPools: { isVisible: false },
+          about: {
+            isVisible: true,
+            editUrl: `${directIntakePaths.aboutMe(
+              poolCandidateId,
+            )}?application=${poolCandidateId}`,
```

Comment:
Can we be more specific with this option name?  Maybe `applicationId`?  Also, the option should probably be concatened to the URL by the URL builder logic in the *Routes.ts file instead of right here.

Reply:
Fixed by [Refactor jobtitle and navigational messages in review application page and edit pages](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3796/commits/6284d286bc06ad582b51d5da87f5d4f8184c308d)

Reply:
Also fixed by [Change parameter name to applicationId](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3796/commits/50d099c26f47191a5858b6858e1d07f88126a9e4)